edx_sailthru/sailthru_content/services/sailthru_api_service.py

```diff
@@ -80,7 +82,7 @@ def clear(self):
                     response.json['error'],
                     response.json['errormsg']
                 )
-                return
+                raise RuntimeError('Failed to connect with Sailthru API')
```

Comment:
I'd recommend asking @schenedx to take a look at these sailthru changes, as I'm not familiar with these scripts.

Reply:
sure.